### PR TITLE
Seed/GBR-02 - api helpers

### PR DESCRIPTION
**Pull request by fablet**
#### Definition of Done:
- [ ] Is there appropriate test coverage? (e.g. ChefSpec, Mocha/Chai, Python, etc.)
- [ ] Does this PR require a Selenium test? (e.g. Browser-specific bugs or complicated UI bugs)
- [ ] Does this PR require a regression test? All fixes require a regression test.
- [ ] Does this add new dependencies? If so, does PIP, npm, bower requirements need to be updated?

### CODE REVIEW

File: seed/models/properties.py [hunk 1]
```diff
@@ -236,12 +238,21 @@ def to_dict(self, fields=None, include_related_data=True):
...
 
+    @property
+    def normalized_address(self):
+        """Ensure normalized address is always available"""
+        if not self._normalized_address and self.address_line_1:
+            self._normalized_address = normalize_address_str(
+                self.address_line_1
+            )
+        return self._normalized_address
+
```
Comment (tallus) [inline seed/models/properties.py]: Does this belong in this PR? I thought this was from upstream.

File: seed/views/api.py [hunk 2]
```diff
@@ -1,18 +1,51 @@
...
+#         generator = SchemaGenerator(
+#             title='SEED Platform API v2',
+#             url='/api/v2/',
+#             urlconf='api.urls'
+#         )
+#         schema = generator.get_schema(request=request)
+#
+#         return Response(schema)
+
+
```
Comment (tallus) [inline seed/views/api.py]: remove commented out code

File: seed/models/properties.py (hunk 1 above)
Comment (fablet) [inline seed/models/properties.py]: I think the normalized_address_str function is implemented upstream, but you altered the fieldname and added the @property method

**Comment by fablet** [comment]
Ready for pull request against upstream SEED

File: seed/lib/superperms/orgs/permissions.py [hunk 3]
```diff
@@ -36,7 +36,13 @@ def is_authenticated(user):
 
 def get_org_or_id(dictlike):
     """Get value of organization or organization_id"""
-    return dictlike.get('organization', dictlike.get('organization_id'))
+    org_query_strings = ['organization', 'organization_id', 'org_id', 'org']
```
Comment (tallus) [inline seed/lib/superperms/orgs/permissions.py]: add a comment to document why this is necessary

**Pull request closed without merging**